QMChatService/QMChatService/Categories/QBChatMessage+QMCustomParameters.h

```diff
@@ -31,6 +31,8 @@
 @property (strong, nonatomic) NSString *dialogPhoto;
 @property (strong, nonatomic) NSDate *dialogUpdatedAt;
 
+@property (strong, nonatomic) NSString *stickerMessage;
```

Comment:
Is this sticker ID string? You need to serialize/deserialize it, see [implementation](https://github.com/QuickBlox/q-municate-services-ios/blob/25beeebd951476b325f4c07e9f103932fd180774/QMChatService/QMChatService/Categories/QBChatMessage%2BQMCustomParameters.m) of already existent properties.